src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,72 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int NAME_INDEX = 0;
+    private static final int VALUE_INDEX = 1;
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        File fileToRead = new File(fromFileName);
+        File fileToWrite = new File(toFileName);
+        writeToFile(fileToWrite, fileToRead);
```

Comment:
it is a bit strange that `writeToFile()` calls `buildInputToFile()`, and `buildInputToFile()` calls `amountOfOneType()` and so on

let's make the methods independent of each other, we will call them one by one in `getStatistic()` method